Source/WebCore/platform/graphics/filters/FEDropShadow.cpp

```diff
@@ -126,6 +126,25 @@ std::unique_ptr<FilterEffectApplier> FEDropShadow::createSoftwareApplier() const
     return FilterEffectApplier::create<FEDropShadowSoftwareApplier>(*this);
 }
 
+OptionSet<FilterMode> FEDropShadow::supportedFilterModes() const
+{
+    return {
+#if PLATFORM(COCOA)
```

Comment:
Again, wherever possible, not using PLATFORM() macros, but instead using scoped ones is preferred as it is better self documenting and provides an easy way to search for places where it is needed.

Reply:
Done.